index.js

```diff
@@ -161,6 +150,22 @@ async function smartContractInitialization() {
     rate = await othertokenExchange.getTokenToEthInputPrice(ethwei)
     log(`1 OTHERtoken buys ${formatEther(rate)} ETH`)
 
+
+
+    const EEwaitms = 60000
+    log("Getting products from E&E")
```

Comment:
You can successfully add smart contracts but fail the import of products which will make the smart-contract-init run again.
I would put this on the top again because it is a dirty check if EE is running.